src/platforms/common/performance/instrumentation/opentelemetry.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: OpenTelemetry Support
```

Comment:
You need to do a couple other things to get this page to be displayed I think. See [this PR](https://github.com/getsentry/sentry-docs/pull/5447/files#top) for help. Also, please add a dash in this file name to make it easier for humans to read later (i.e., open-telemetry). I'd definitely like to see this again once it's able to be previewed on Vercel.

Reply:
The preview is working now: https://sentry-docs-git-abhi-otel-node-docs.sentry.dev/platforms/node/performance/instrumentation/opentelemetry/

Reply:
Note that we always use `opentelemetry` without a dash, as it is `OpenTelemetry` originally.

Reply:
gotcha, okay